teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super
+    @specialization = specialization
```

Comment:
- [ ] When creating a teacher, kindly add the `age`, `name`, `parent_permission = true` in the parameters. 
- [ ] Also, when calling super, kindly add the parameters `age`, `name`, `parent_permission` to the super method.